src/app/services/contact.service.ts

```diff
@@ -0,0 +1,20 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs/Observable';
+
+import { Datastore } from './datastore.service';
+import { Contact } from '../models/contact';
+
+@Injectable()
+export class ContactService {
+
+  constructor(private datastore: Datastore) { }
+
+  getAll(): Observable<Contact[]> {
+    return this.datastore.findAll(Contact, {})
+             .map(res => res.getModels());
```

Comment:
Using `.map()` here which will return an array of all contacts. With this, there's no need for the `toIterable()` function